pkgs/tools/games/pokefinder/default.nix

```diff
@@ -1,45 +1,72 @@
 { lib
 , stdenv
+, copyDesktopItems
+, makeDesktopItem
 , fetchFromGitHub
 , cmake
 , qtbase
 , qttools
 , qtwayland
+, imagemagick
 , wrapQtAppsHook
 , gitUpdater
 }:
 
 stdenv.mkDerivation rec {
   pname = "pokefinder";
-  version = "4.0.1";
+  version = "4.1.1";
 
   src = fetchFromGitHub {
     owner = "Admiral-Fish";
     repo = "PokeFinder";
     rev = "v${version}";
-    sha256 = "j7xgjNF8NWLFVPNItWcFM5WL8yPxgHxVX00x7lt45WI=";
+    sha256 = "fYBeWc9eYLbj4+ku1jwaO5ISL8a7WJnBHJ4qz4W8RHA=";
     fetchSubmodules = true;
+    # the repo has identical cmake and CMake folders, causing issues on macOS
+    postFetch = if stdenv.isDarwin then ''
+    mv $out/cmake $out/cmake.tmp
+    mv $out/cmake.tmp $out/CMake
+    '' else ''
+    rm -rf $out/cmake
```

Comment:
This cannot be conditional as it influences the hash.

Reply:
It does not (I've confirmed this). The issue is that 4.1.0-4.1.1 accidentally include identical `cmake` and `CMake` folders, while on macOS `/nix/store` is on a case-insensitive filesystem (leading to only one of the two being created). After `postFetch` is run, both Linux and macOS have just the `CMake` folder.